### ISSUE TITLE
Alpha status?

### ISSUE THREAD

**Issue by Havunen** [Oskarsson/AspNetCore.JsonPatch#1]
Hey!

Cool project! I was planning to do this same thing and then found this project! :) 

I noticed the project is still in alpha state, are there known issues or something what needs to be fixed before stable release?

**Comment by coderdnewbie** [comment]
I agree this would be very useful as Microsoft will not do it. What is the latest status?
